test/e2e/config_amd64.go

```diff
@@ -1,15 +1,15 @@
-	BB_GLIBC                 = "docker.io/library/busybox:glibc"                                                                                                                         //nolint:revive,stylecheck
-	REGISTRY_IMAGE           = "quay.io/libpod/registry:2.8.2"                                                                                                                           //nolint:revive,stylecheck
-	CITEST_IMAGE             = "quay.io/libpod/testimage:20221018"                                                                                                                       //nolint:revive,stylecheck
-	SYSTEMD_IMAGE            = "quay.io/libpod/systemd-image:20230106"                                                                                                                   //nolint:revive,stylecheck
-	CIRROS_IMAGE             = "quay.io/libpod/cirros:latest"                                                                                                                            //nolint:revive,stylecheck
+	STORAGE_FS               = "overlay"                                                                                                                                  //nolint:revive,stylecheck
+	STORAGE_OPTIONS          = "--storage-driver overlay"                                                                                                                 //nolint:revive,stylecheck
+	ROOTLESS_STORAGE_FS      = "overlay"                                                                                                                                  //nolint:revive,stylecheck
+	ROOTLESS_STORAGE_OPTIONS = "--storage-driver overlay"                                                                                                                 //nolint:revive,stylecheck
+	CACHE_IMAGES             = []string{ALPINE, BB, NGINX_IMAGE, REDIS_IMAGE, REGISTRY_IMAGE, INFRA_IMAGE, CITEST_IMAGE, HEALTHCHECK_IMAGE, SYSTEMD_IMAGE, fedoraToolbox} //nolint:revive,stylecheck
+	NGINX_IMAGE              = "quay.io/libpod/alpine_nginx:latest"                                                                                                       //nolint:revive,stylecheck
+	BB_GLIBC                 = "docker.io/library/busybox:glibc"                                                                                                          //nolint:revive,stylecheck
+	REGISTRY_IMAGE           = "quay.io/libpod/registry:2.8.2"                                                                                                            //nolint:revive,stylecheck
+	CITEST_IMAGE             = "quay.io/libpod/testimage:20240123"                                                                                                        //nolint:revive,stylecheck
+	SYSTEMD_IMAGE            = "quay.io/libpod/systemd-image:20240124"                                                                                                    //nolint:revive,stylecheck
```

Comment:
For future: is there a good reason for duplicatingly defining `CITEST_IMAGE` and others in `config_TAGGED.go` instead of just `config.go`? Maybe historically these were different? I am absolutely not going to change that in this already-too-big PR, but I'd like to consider doing so in a followup.